examples/benchmark.py

```diff
@@ -52,8 +52,8 @@
 
     devito = parser.add_argument_group("Devito")
     devito.add_argument("-dse", default="advanced", nargs="*",
-                        choices=["noop", "basic", "factorize", "approx-trigonometry",
-                                 "glicm", "advanced", "speculative"],
+                        choices=["noop", "basic", "advanced", "speculative",
+                                 "aggressive"],
```

Comment:
I realise these are in flux, but we will need some serious developer documentation for all these modes soon. Just a reminder though, not as part of this PR.